apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/NewCodeProjectTypeStep.ts

```diff
@@ -0,0 +1,125 @@
+    await fs.ensureDir(logicAppFolderPath);
+
+    // Set the Logic App and Functions folder paths as properties of the AzureWizard
+    context.logicAppFolderPath = logicAppFolderPath;
+    context.functionFolderPath = functionFolderPath;
+
+    // Generate the VS Code configuration files in the specified folder
+    const createDesignerConfigFiles = new DesignerConfig();
+    await createDesignerConfigFiles.prompt(context);
+
+    // Create the workspace file
+    await this.createWorkspaceFile(context);
+  }
+
+  /**
+   * Determines whether the user should be prompted to select a project type.
+   * @param context The project wizard context.
+   * @returns True if the user has not yet selected a project type or if the workflowProjectType is WorkflowProjectType.Bundle, false otherwise.
+   */
+  public shouldPrompt(context: IProjectWizardContext): boolean {
+    return context.workflowProjectType === undefined || context.workflowProjectType === WorkflowProjectType.Bundle;
+  }
+
+  /**
+   * Creates a sub-wizard that will be used to create the project.
+   * @param context The project wizard context.
+   * @returns An object containing the prompt and execute steps for the sub-wizard.
+   */
+  public async getSubWizard(context: IProjectWizardContext): Promise<IWizardOptions<IProjectWizardContext>> {
+    const executeSteps: AzureWizardExecuteStep<IProjectWizardContext>[] = [];
+
+    const promptSteps: AzureWizardPromptStep<IProjectWizardContext>[] = [];
+    const projectType: WorkflowProjectType = nonNullProp(context, 'workflowProjectType');
+
+    // Set the workflow project path based on the project type
+    let workflowProjectPath: string;
+    if (projectType === WorkflowProjectType.Functions) {
+      workflowProjectPath = context.functionFolderPath;
+    } else {
+      workflowProjectPath = path.join(context.projectPath, context.workspaceName, 'LogicApp');
+    }
+
+    executeSteps.push(new WorkflowCodeProjectCreateStep(workflowProjectPath));
+
+    // Add any necessary steps to initialize the project for VS Code
+    await addInitVSCodeSteps(context, executeSteps);
+
+    // Create the sub-wizard options object
+    const wizardOptions: IWizardOptions<IProjectWizardContext> = { promptSteps, executeSteps };
+
+    // Add any necessary prompt steps to the sub-wizard
+    promptSteps.push(new InvokeFunctionProjectSetup());
+    promptSteps.push(
+      await CodeProjectWorkflowStateTypeStep.create(context, {
+        isProjectWizard: true,
+        templateId: this.templateId,
+        triggerSettings: this.functionSettings,
+      })
+    );
+
+    return wizardOptions;
+  }
+
+  private async createWorkspaceFile(context: IProjectWizardContext): Promise<void> {
+    const projectPath = path.join(context.projectPath, context.workspaceName);
+    const workspaceFilePath = path.join(projectPath, `${context.workspaceName}.code-workspace`);
+    const workspaceData = {
+      folders: [
+        {
+          name: 'Function',
```

Comment:
Can we clarify this is "Azure Functions" and not just any "functions", for example, `toString()`?

```suggestion
          name: 'Azure Functions',
```

Reply:
> Can we clarify this is "Azure Functions" and not just any "functions", for example, `toString()`?

@rohithah is this something you would like to do now or save for the backlog? We only support .NET frameworks right now and from past experience implying full functionality of azure functions may be a bit confusing. Let me know your thoughts.

Reply:
@ecfan, @samikay101   - I think using "Azure Functions" would be confusing since this cannot directly be deployed to an azure functions. As for full functionality, given that we plan on adding other .Net framworks in the future, I think its ok as the project structure would remain the same when we add that functionality.

Reply:
also note that if we change the fodler path, we need to update the csproj file to reflect the new name so that bin placing works correctly

Reply:
> 

it sounds like we're all on the same page, and yes it is possible to remove the space in the folder Name, we can use camelCase naming.